docs/preview/features/powershell/azure-security.md

```diff
@@ -81,3 +81,22 @@ PS> Write-Host "Current SubscriptionId:" $Global:subscriptionId
 PS> Write-Host "Current AccessToken:" $Global:accessToken
 # Current AccessToken: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
 ```
+
+## Granting a resource access to all resources within a specific resource group
+
+In some cases a resource needs to be granted access to all resources present within a specific resource group.
+
+| Parameter                 | Mandatory  | Description                                                                                  |
+| ------------------------- | ---------- | -------------------------------------------------------------------------------------------- |
+| `TargetResourceGroupName` | yes        | The name of the resource group to which access should be granted.                            |
+| `ResourceGroupName`       | yes        | The name of the resource which should be granted access.                                     |
```

Comment:
It seems the description for these 2 parameters (ResourceGroupName and ResourceName) has been switched around.

Reply:
Ah, yes, sorry. Good catch!

Reply:
Done that. Resolving.